src/store/actions/contractActions.ts

```diff
@@ -0,0 +1,22 @@
+import axios from "../../axios/axiosinst"
+
+import { GET_ALL_CONTRACTS } from "./actionTypes"
+
+import { ContractType } from "../../utility/types"
+
+//This function formats the action for the reducer to put into state
+const assembleAllContracts = (data: Array<ContractType>) => {
+  return { type: GET_ALL_CONTRACTS, payload: { contractData: data } }
+}
+
+// Here, hash is an argument given when this action is called.
+//It is called with this function, since it has extra processes to do first.
+export const getAllContracts = (channelHash: string) => {
+  return (dispatch: any) => {
+    axios.get(`/chaincode/${channelHash}`).then((response) => {
+      console.log("TEST CONTRACT GET: ", response.data.chaincode)
```

Comment:
remove please

Reply:
This console.log is still in use for developing that page, I will keep this for now